resources/articles/oauth/register.md

```diff
@@ -0,0 +1,75 @@
+:title POST /oauth/introspect
+
+:aside
+
+## See also
+
+* [POST /oauth/token](/oauth/token/)
+* [POST /oauth/ro](/oauth/ro/)
+* [GET /oauth/userinfo](/oauth/userinfo/)
+* [GET /oauth/jwks](/oauth/jwks/)
+* [GET /oauth/authorize](/oauth/authorize/)
+* [Server to Server Auth](/s2s-auth/)
+* [Token introspection](/token-introspection/)
+
+:body
+
+The `/oauth/register` endpoint is used to register a new client under a merchant. The use case
+for this endpoint is mainly dynamically registered clients for s2s auth.
+Dynamic client registration is defined in [RFC7591](https://tools.ietf.org/html/rfc7591).
+
+The bearer token in the request must have scope `spid:merchants/<merchant>|write` and token `client_id` must belong to that `<merchant>`.
```

Comment:
"The bearer token in the request must have scope `spid:merchants/<merchant>|write` and token `client_id` must belong to that `<merchant>`." -> "The bearer token in the request must contain scope `spid:merchants/<merchant>|write`" and be issued to a client belonging to `<merchant>`.